lib/OpenQA/Worker/Jobs.pm

```diff
@@ -260,7 +260,9 @@ sub upload {
             $hosts->{$current_host}{ua}->post($ua_url => form => {temporary => $res->res->json->{temporary}});
         }
         else {
+            open(my $log, '>>', "autoinst-log.txt");
             print $log "Checksum/size comparison of $filename FAILED\n";
+            close $log;
```

Comment:
I would remove this close, and  the one on [line 256](https://github.com/mnowaksuse/openQA/blob/b4eb569ce2f08ee7f2215040f74eb3ceb08071a1/lib/OpenQA/Worker/Jobs.pm#L256) as the file descriptor will be closed automatically when the variable goes out of scope.